include/deal.II/multigrid/multigrid.h

```diff
@@ -99,9 +99,8 @@ class Multigrid : public Subscriptor
             Cycle                              cycle = v_cycle);
 
   /**
-   * Experimental constructor for cases in which no DoFHandler is available.
-   *
-   * @warning Not intended for general use.
+   * Constructor. Same as above but you can determine the multigrid
+   * levels to use yourself.
```

Comment:
why would one need this constructor at all? What's the usage case?
It's possible to override `minlevel` and `maxlevel` using corresponding methods while using the constructor with dof_handler.